fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/api/CommitOption.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.persistence.ocfl.api;
+
+
+/**
+ * Options for defining the behavior when performing a commit to an OCFL object
+ *
+ * @author bbpennel
+ */
+public enum CommitOption {
+    TO_MUTABLE_HEAD,
```

Comment:
I think this could just be MUTABLE_HEAD...?

Reply:
I agree, these two options should have a consistent naming scheme. I've updated it.